### ISSUE TITLE
Image doesn't download anything

### ISSUE THREAD

**Issue by axelcypher** [murzart/PerfectHeist2#1]
I've tried to set up this Image as described, with no luck. When i ssh into docker the server folder is empty, and i cant start the container because, obviously, PerfectHeist2Server.sh is missing. 

Any idea why this happens?

**Comment by murzart** [comment]
Hey, I just tested it and I was not able to reproduce your issue. Please ensure that you are using the latest version of the image and there is enough space on your system, since it's downloading the game during startup.

If that does not help, please provide your from `docker logs ph2-dedicated`

**Comment by axelcypher** [comment]
Yeah, i've used the latest version. There's also plenty of space left. I also checked the permissions, everything should work...
The Docker Image i created myself works fine (I'm too lazy and have not enough time to wirte it properly, so i hardcoded my settings into it), I'm just struggeling with Custom Maps. But in long term i want to use your image, as it is way more flexible than mine.  

Will provide the logs asap.
